src/lib/auth.ts

```diff
@@ -502,6 +514,16 @@ export const createFakeAuthentication = (options: CreateFakeAuthenticationOption
     students: [],
     teachers: [],
   };
+  // Add the random student to the class first and then fill the class
+  classInfo.students.push(
+    createFakeUser({
+      appMode,
+      classId,
+      userType: "student",
+      userId: `${userId}`,
```

Comment:
If the URL included `student:10`, wouldn't this create the student twice?  Or does `createFakeUser` make sure that doesn't happen?